org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/dashboard/EventWindow.java

```diff
@@ -167,7 +168,7 @@ public void onEvent(Event e) throws Exception {
 			setVisible(false);
 		else if (e.getTarget() == confirmPanel.getButton(ConfirmPanel.A_ZOOM)) {
 			if (R_Request_ID > 0)
-				AEnv.zoom(417, R_Request_ID);
+				AEnv.zoom(SystemIDs.TABLE_R_REQUEST, R_Request_ID);

```

Comment:
why not just use the I_R_Request.Table_ID constant instead ?

Reply:
Done, and changed others that I have found.